styles/utilities.css

```diff
@@ -13,3 +13,15 @@
   white-space: nowrap;
   border-width: 0;
 }
+
+.button {
+  color: #ffffff;
+  background-color: #f24b88 ;
+  text-align: center;
+  border-radius: 10px;
+  text-decoration: none;
+  padding: 10px;
+  text-transform: uppercase;
+  font-size: 0.813rem;
```

Comment:
This looks like a value from Figma. In real live, you would discuss this with the designer a choose a more sane value.